ci/datamgr.py

```diff
@@ -532,7 +530,6 @@ def bigquery(data_directory, ignore_missing_dependency, **params):
     # Load an ingestion time partitioned table.
     functional_alltypes_path = data_directory / 'functional_alltypes.csv'
```

Comment:
This is duplicated from above, isn't it?

Reply:
removed in https://github.com/ibis-project/ibis/commit/04b773a40b5317b46b0a8aed939e488ac9f05213